examples/ios/sdk-cubic/sdk_cubic.h

```diff
@@ -0,0 +1,19 @@
+//
+//  sdk_cubic.h
```

Comment:
Since the sdk code was moved to the grpc directory, can we remove this file (and the Info.plist file) from here?